packages/components/psammead-byline/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,137 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Byline should render correctly 1`] = `
+.c0 {
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  margin-right: -0.5rem;
+  margin-left: -0.5rem;
+}
+
+.c1 {
+  border-radius: 50%;
+  height: 4rem;
+  margin-right: 0.5rem;
+  margin-left: 0.5rem;
+  width: 4rem;
+}
+
+.c2 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+}
+
+.c3 {
+  color: #B80000;
+}
+
+.c4 {
+  color: #5A5A5A;
+}
+
+@media (min-width:37.5rem) {
+  .c2 {
+    font-size: 0.875rem;
+  }
+}
+
+<div
+  class="c0"
+>
+  <img
+    class="c1"
```

Comment:
missing alt=""??

Reply:
This is correct, because `alt` is unset in this test. I will add another case where it isn't.

Reply:
Good spot! I'll fix that.

Reply:
thanks